OpenRCT2.xcodeproj/project.pbxproj

```diff
@@ -2559,7 +2561,7 @@
 				F76C838C1EC4E7CC00FA49E2 /* MemoryStream.cpp */,
 				F76C838D1EC4E7CC00FA49E2 /* MemoryStream.h */,
 				2ADE2F24224418B2002598AF /* Meta.hpp */,
-				F76C838E1EC4E7CC00FA49E2 /* Nullable.hpp */,
+				F76C838E1EC4E7CC00FA49E2,
```

Comment:
Stray change

Reply:
this is done automatically by @ZehMatt 's tool. Not sure why it did that. I'll manually readd it.